app/create/page.tsx

```diff
@@ -1,117 +1,11 @@
-"use client";
+import GameForm from "@/components/GameForm";
+import React from "react";
 
-import { Button, Stack, TextField, Autocomplete } from "@mui/material";
-import Link from "next/link";
+export const metadata = {
+  title: "Game Trucker",
+  description: "Game Trucker",
+};
```

Comment:
meta title and description should give an context of the page.
In this case it would be something like:
```suggestion
export const metadata = {
  title: "Create game tracking",
  description: "Here you can add a tracking for your game",
};
```

To help you understand better what the meta tags are: https://moz.com/learn/seo/meta-description